uthdev/mock-shop
# #171171948 Setup progreSQL with sequelize ORM

#### Description of Task to be completed?
- Setup Sequelize ORM with PostgreSQL
- Create models and migrations structure
- setup environment variables
- update package.json scripts
- setup user model
- setup cart model
- setup product model
- add unit test for the models
 #### How should this be manually tested?
- clone the repo
- cd into the root folder of the cloned repo
- run `npm install`  to install necessary dependencies 
- open the terminal and type `touch .env` in the terminal
- ensure you have Postgres installed and setup
- create your database type  `createdb <db-name>`
- open the .env file and put your database info inside, check out the .env-example file in the root of the repo to see how to do this
- run `npm run migrate` to create your tables
- run `npm test` to test the models
 #### What are the relevant pivotal tracker stories?
[ #171171948](urlhttps://www.pivotaltracker.com/story/show/171171948)
 #### Screenshots (if appropriate)
None
 #### Questions:
None
